### PR TITLE
Bump BSI for the d2l-dropdown and d2l-menu lit migration

### PR DESCRIPTION
**Pull request by AllanKerr**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -976,9 +976,9 @@
       }
     },
     "@brightspace-ui-labs/edit-in-place": {
-      "version": "1.0.10",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui-labs/edit-in-place/-/edit-in-place-1.0.10.tgz",
-      "integrity": "sha512-uV0MoPs2XkMKYX3kSgEieUZBaTqV4iodN6P3U+IlXLxNPlhm/EulodqP+nQ3jbTpA7/PuT2Ka0AYBM4L8MWYbQ==",
+      "version": "1.0.11",
```
Comment (AllanKerr) [inline package-lock.json]: @BhamelinD2L

File: package-lock.json [hunk 2]
```diff
@@ -4123,16 +4140,6 @@
...
-    "d2l-hierarchical-view": {
-      "version": "github:BrightspaceUI/hierarchical-view#0935fd62fd9a444a9a11ac4f2501e2d5af9e56d7",
-      "from": "github:BrightspaceUI/hierarchical-view#semver:^2",
-      "dev": true,
-      "requires": {
-        "@brightspace-ui/core": "^1.7.1",
-        "@polymer/polymer": "^3",
-        "fastdom": "^1.0.8"
-      }
-    },
```
Comment (dlockhart) [inline package-lock.json]: 🎉

File: package-lock.json [hunk 3]
```diff
@@ -3727,7 +3745,7 @@
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#c6abf39ec53043e636181cf471391345ad04ad6f",
+      "version": "github:BrightspaceHypermediaComponents/activities#f8cef018d2ec20382ac491db91fd43597c7d5ce9",
```
Comment (AllanKerr) [inline package-lock.json]: @BhamelinD2L

File: package-lock.json [hunk 4]
```diff
@@ -4600,7 +4603,7 @@
       }
     },
     "d2l-sequences": {
-      "version": "github:BrightspaceHypermediaComponents/sequences#cf455c740e9dc229049c52bb9f2b5647d1d3e347",
+      "version": "github:BrightspaceHypermediaComponents/sequences#7e893c7cade327ab5f4ccb6ac86d8472c2c386fc",
```
Comment (AllanKerr) [inline package-lock.json]: @sarahboyd

File: package-lock.json [hunk 5]
```diff
@@ -25906,7 +25909,7 @@
       "dev": true
     },
     "siren-sdk": {
-      "version": "github:BrightspaceHypermediaComponents/siren-sdk#5ead7c2e95a9f16a077afcaaaad4691146214ab0",
+      "version": "github:BrightspaceHypermediaComponents/siren-sdk#c43d002369979367874ed1a6c09624c5c21f495b",
```
Comment (AllanKerr) [inline package-lock.json]: @emil-furniss

Comment (AllanKerr) [reply package-lock.json]: Had to resolve conflicts, this got bumped in [the preceding PR](https://github.com/Brightspace/brightspace-integration/pull/1900/files).

**Pull request merged**